pkg/cmd/workspace/util/repository_wizard.go

```diff
@@ -42,20 +42,23 @@ func getRepositoryFromWizard(config RepositoryWizardConfig) (*apiclient.GitRepos
 	}
 
 	if (len(config.UserGitProviders) == 0 && len(samples) == 0) || config.Manual {
-		return create.GetRepositoryFromUrlInput(config.MultiProject, config.ProjectOrder, config.ApiClient, config.SelectedRepos)
+		repo, err := create.GetRepositoryFromUrlInput(config.MultiProject, config.ProjectOrder, config.ApiClient, config.SelectedRepos)
```

Comment:
![Screenshot 2024-09-23 at 09 41 30](https://github.com/user-attachments/assets/19f2d334-5326-414d-afaf-ac9e49a7e043)

Not sure where to put this, but personal projects on gitlab don't work anymore.